api/docs/code_reviews.dox

```diff
@@ -215,7 +216,10 @@ separate commit (these separate commits will all be squashed together upon mergi
 When the requested changes have been pushed, request a re-review from the reviewer so they know that the pull request is ready for another round of reviewing.
 This can be done by clicking the re-review button next to the reviewer's name at the top of the right sidebar on the pull request page.
 
-The final squash-and-merge step will squash these additional commits with the original to make a single commit that will be fast-forward-merged into master (see below for more details).
+The final squash-and-merge step will squash these additional commits with the
+original to make a single commit that will be fast-forward-merged into master (see
+below for more details).  The final merge commit's title and description come
```

Comment:
I've also noted Github automatically inserting some newlines in the commit message automatically to limit line length. Worth pointing out that that doesn't need to be done manually (anymore)?

Reply:
Hmm, was not aware of that: do you have an example or github docs showing what they do?

Reply:
See the desc of PR #7124 which has long lines, but the corresponding commit doesn't: 73aae264aa7a41bf930b110572ea8f91add93d1c

Reply:
Counterexample: https://github.com/DynamoRIO/dynamorio/pull/7097 and https://github.com/DynamoRIO/dynamorio/commit/b0237d21570b92a7d11a2b7a3edf4d25c5c0e1c8.
Not seeing info on a first search for github deliberately doing something here; so since we don't understand it going to not mention it here at this time.

Reply:
Do you have a browser extension affecting text edit boxes, maybe?  Maybe that's what wrapped yours when you clicked on merge?

Reply:
Hmm, none that I'm aware of... 

> so since we don't understand it going to not mention it here at this time.

ack.

Reply:
FTR, I just did a merge on the Github Android app. It inserted the newlines as I mentioned previously. E.g., 4adbf0f7d50c723d80ea16417fa3a1f25ecf2772 and https://github.com/DynamoRIO/dynamorio/pull/7158